01week/rockPaperScissors.js

```diff
@@ -7,17 +7,40 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+const rockPaperScissors=(hand1, hand2)=> {
+  const aTie = "Its's a tie!";
+  const oneWins = "Hand one wins!";
+  const twoWins = "Hand two wins!";
 
-function rockPaperScissors(hand1, hand2) {
+  if (hand1 === hand2) {
```

Comment:
Before comparing the two values you need to convert them to uniform strings so you don't throw errors or bad values. For instance try passing "silly" and "scissors" as arguments.